plugins/dev-create/bin/app.js

```diff
@@ -0,0 +1,64 @@
+const chalk = require('chalk');
+const fs = require('fs-extra');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const {createDir} = require('./common');
+
+exports.createApp = function(name, options) {
+  if (options.typescript) {
+    console.error('Not yet implemented');
+    process.exit(1);
+  }
+
+  const root = process.cwd();
+  const dir = options.dir ?? name;
+  const appPath = path.join(root, dir);
+
+  // Create the new directory for the app if needed.
+  createDir(appPath);
+  console.log(
+      `Creating a new Blockly application called ${name} in ${appPath}`);
+
+  // Copy over files from sample-app directory.
+  const sampleDir = '../../../examples/sample-app';
+  fs.copySync(path.resolve(__dirname, sampleDir), appPath, {
+    filter(src) {
+      const excludes = [
+        'node_modules', 'dist', 'package-lock.json', 'package.json'];
+      for (const file of excludes) {
+        if (src.indexOf(file) > -1) return false;
+      }
+      return true;
+    },
```

Comment:
My reading of `fs-extra`'s `copySync` is that [the filter returning false prunes the entire subtree](https://github.com/jprichardson/node-fs-extra/blob/f3a7f0beeb5858c628b10010ad819c813c7f3565/lib/copy/copy-sync.js#L29) so I think this whole section can be simplified:
```suggestion
    filter(src) {
      return !(['node_modules', 'dist', 'package-lock.json', 'package.json'].includes(file));
    },
```

If for some reason that doesn't work then I think:
* The formatting of `excludes` could be checked (did it get `clang-format`ted?)
* You should use `.includes` instead of `.indexOf`.